bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-spoor",
-  "version": "3.10.1",
+  "version": "3.10.2",
   "framework": ">=5.5",
```

Comment:
Hi Chris, You will need to bump the FW dependency to include your Core additions

Reply:
Thanks @cahirodoherty-learningpool, should I bump package.json in adapt-contrib-core to 6.0.5 and then bump the FW dependency here to v5.17.7?

Reply:
And let @joe-allen-89 know so he's aware for releases